package.json

```diff
@@ -24,28 +24,30 @@
   },
   "homepage": "https://github.com/pinterest/esprint#readme",
   "dependencies": {
-    "dnode": "^1.2.2",
-    "fb-watchman": "^2.0.0",
-    "glob": "^7.1.1",
-    "sane": "^1.6.0",
-    "worker-farm": "^1.3.1",
-    "yargs": "^8.0.1"
+    "dnode": "https://github.com/christianvuerings/dnode/#e08e620b18c9086d47fe68e08328b19465c62fb7",
```

Comment:
Forked `dnode` since https://github.com/substack/dnode/pull/194 is not yet merged